app/base/static/assets/js/vis/simple-bar-chart.js

```diff
@@ -1,6 +1,10 @@
 class SimpleBarChart {
+    /*
     CHART_WIDTH = 1000;
     CHART_HEIGHT = 600;
+    */
+    CHART_WIDTH = window.innerWidth - 260;//side bar width is 260
```

Comment:
Also if the number 260 is similar for all of the charts then we should define the number in common.js. We should try to avoid magic numbers.

Reply:
260 is the side Manu bar width,  however, in the my latest code I use 
CHART_WIDTH = document.getElementById('charts').offsetWidth